src/main/java/com/javachallengers/simpson/exception/BusinessException.java

```diff
@@ -0,0 +1,18 @@
+package com.javachallengers.simpson.exception;
+
+public class BusinessException extends Exception {
```

Comment:
Java has a lot of unchecked exception, is necessary create a new one? I'm asking this because could we use IllegalStateException and so on, in order to avoid force the client a try/catch block. It's just a suggestion.

Reply:
I create this to separate business rules exceptions (checked) from internal server error exceptions (unchecked). So it can be used in controller advices.